### DISCUSSION TITLE
hook function not trigger

### DISCUSSION THREAD

**Discussion by dengsong666** [nuxt/content#2613] [Q&A]
I want to know what happens when content:file:beforeParse is triggered, write some code according to the official documentation, when I start the project and access my markdown file and I don't see the print-out beforeParse, what do I need to do extra?

```
export default defineNitroPlugin((nitroApp) => {
  nitroApp.hooks.hook('content:file:beforeParse', (file) => {
    console.log("beforeParse")
  })
})
```

**Comment by AloisSeckar** [comment]
I was just about to ask the same thing. The plugin itself gets initialited, but the console log for this hook never happens in my app 😕 

When exactly shoud `'content:file:beforeParse'` happen?
